src/components/Form/FetchDocumentForm.jsx

```diff
@@ -67,11 +67,11 @@ const FetchDocumentForm = () => {
       defaultMessage="To be searched..."
     >
       <Box sx={{ minWidth: 120 }}>
-        <form onSubmit={handleGetDocumentSubmission} autoComplete="off">
+        <form onSubmit={handleGetDocumentSubmission}>
           <InputLabel id="search-doctype">
             <i>Select Document Type</i>
           </InputLabel>
-          <FormControl fullWidth variant="standard">
+          <FormControl  variant="standard">
```

Comment:
Prettier caught the extra space here. do a `npm run prettier:run`.

Reply:
When I run that I get this error:
```
[error] No files matching the pattern were found: "'src/**/*.js'".
[error] No files matching the pattern were found: "'src/**/*.jsx'".
[error] No files matching the pattern were found: "'test/**/*.js'".
[error] No files matching the pattern were found: "'test/**/*.jsx'".
```

Reply:
Not sure what happened, but this is what I get when I ran your latest changes.

<img width="1440" alt="Screen Shot 2023-05-24 at 23 09 27" src="https://github.com/codeforpdx/PASS/assets/14917816/baf1c0f0-3083-4a1c-8843-2ead3f9cf4de">

Reply:
Still thinking about what might have gone wrong on your end with Prettier, but the space problem is gone, so I'm fine for now.

Reply:
Doesn't seem to be a real issue, but I wanted to provide more information on this in case it's useful to anyone.

Running `npm run prettier:run` yields the same error.

However, when I run `npm run prettier:check`, I get this: 

```
Checking formatting...
[error] No files matching the pattern were found: "'src/**/*.js'".
[error] No files matching the pattern were found: "'src/**/*.jsx'".
[error] No files matching the pattern were found: "'test/**/*.js'".
[error] No files matching the pattern were found: "'test/**/*.jsx'".
All matched files use Prettier code style!
```

The only explanation in my mind is that since I have Prettier installed on my editor anyway, the script checks for that and does not execute properly.

Reply:
Also, refrain from closing conversations other people started.